rfcs/0000-template.md

```diff
@@ -0,0 +1,44 @@
+- Start Date: YYYY-MM-DD
+- RFC PR: -
+- Issue: -
+- Affected components
+    + [X] [ui5-builder](https://github.com/SAP/ui5-builder)
```

Comment:
Maybe don't "check" any component by default. If someone forgets to check them it's easier to see when no component is checked.

Reply:
I'm not sure whether people will otherwise know that an "X" needs to be used to "check" a component...

Reply:
You can also check them within the markdown preview just like normal checkboxes.

Reply:
Not in your local IDE

Reply:
Hm okay. What about an HTML comment that describes what to do? It will be visible but is not rendered in the markdown. This way we might add other disclaimers which should not be part of the final document.

Reply:
Great idea 👍

Reply:
Done